libgeopmd/debian/rules

```diff
@@ -15,6 +16,9 @@ override_dh_auto_configure:
 	if [ "$(DEB_BUILD_ARCH)" = "amd64" ] || [ "$(DEB_BUILD_ARCH)" = "arm64" ]; then \
 	    CONFIG_OPTION="--enable-nvml $$CONFIG_OPTION"; \
 	fi; \
+	if [ "$(CODENAME)" = "noble" ] && [ "$(DEB_BUILD_ARCH)" = "amd64" ]; then \
```

Comment:
Not sure the codename check for level zero support is doing much since the build requirements are independent of the release codename.